featurebyte/routes/lazy_app_container.py

```diff
@@ -211,7 +241,7 @@ def get(self, key: str) -> Any:
         -------
         Any
         """
-        return self._get_key(key)
+        return self._handle_block_modification_check(self._get_key(key))
 
     def __getattr__(self, key: str) -> Any:
-        return self._get_key(key)
+        return self._handle_block_modification_check(self._get_key(key))
```

Comment:
Curious why not call `_handle_block_modification_check` in `_get_key()`?

Reply:
Need to modify two places in the `_get_key` (one is retrieving it from the dictionary, another one is to create a new service).